guaranteed-message-delivery/message_consumer.bal

```diff
@@ -0,0 +1,62 @@
+import ballerina/http;
+import ballerina/log;
+import ballerinax/rabbitmq;
+import ballerinax/twilio;
+
+type MessageContent record {|
+    *ReservationRequest;
+    string category;
+|};
+
+type RabbitMqMessage record {|
+    *rabbitmq:AnydataMessage;
+    MessageContent content;
+|};
+
+configurable string fromNumber = ?;
+configurable string accountSId = ?;
+configurable string authToken = ?;
+
+final http:Client hospitalBackend = check new ("http://localhost:9090/");
+
+final twilio:Client twilioEp = check new ({
+    twilioAuth: {
+        accountSId: accountSId,
+        authToken: authToken
+    }
+});
+
+@rabbitmq:ServiceConfig {
+    queueName,
+    autoAck: true
+}
+service rabbitmq:Service on new rabbitmq:Listener(rabbitmq:DEFAULT_HOST, rabbitmq:DEFAULT_PORT) {
```

Comment:
In the README/description, let's make sure to mention what these values are when they are first used.